frontend/src/js/controllers/authCtrl.js

```diff
@@ -55,6 +55,29 @@
             vm.wrnMsg = {};
         }
 
+        vm.passwordChecksignUp = function(password1, password2) {
+            var password1_len = password1.length;
+            var password2_len = password2.length;
+
+            if (password1_len >= 8 && password2_len >= 8)
+            {
+                if (password1 === password2)
+                {
+                    vm.confirmMsg = "Passwords Match !"
```

Comment:
return values of confirmMsg and status(true/false) in a single object as key value pair.